common/src/jni/main/include/conscrypt/bio_input_stream.h

```diff
@@ -14,18 +14,18 @@
  * limitations under the License.
  */
 
-#ifndef CONSCRYPT_BIOINPUTSTREAM_H_
-#define CONSCRYPT_BIOINPUTSTREAM_H_
+#ifndef MAIN_INCLUDE_CONSCRYPT_BIO_INPUT_STREAM_H_
+#define MAIN_INCLUDE_CONSCRYPT_BIO_INPUT_STREAM_H_
 
 #include <jni.h>
 #include <openssl/ssl.h>
 
-#include "BioStream.h"
+#include "conscrypt/bio_stream.h"
 
 namespace conscrypt {
 
 class BioInputStream : public BioStream {
-public:
+ public:
```

Comment:
Per clang-format, this project's style seems to be the old one, not the new one. You should find the Android C++ style guide.